radix-transactions/src/model/executable.rs

```diff
@@ -79,6 +79,31 @@ impl Default for TransactionCostingParameters {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor, ManifestSbor)]
+pub struct TransactionCostingParametersV1 {
```

Comment:
Can we add a note (regular comment) explaining why it is introduced?

(also, maybe in the PR description, some link to the investigation)